code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -1,3 +1,4 @@
+#define CALL_BOT_COOLDOWN 900

```

Comment:
I tried to remove it, it came back, its github's interface being odd. the code looked alright and test merged fine.